eddymotion/model.py

```diff
@@ -53,8 +53,8 @@ def init(gtab, model="TensorModel", **kwargs):
                 "isotropic": ExponentialIsotropicModel,
             }
 
-        elif model.lower().startswith("tensor"):
-            Model = TensorModel
+        elif model.lower().startswith("dti"):
```

Comment:
I guess now we can do:
```suggestion
        elif model.lower() == "dti":
```

(same for DKI, btw)